src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -182,6 +182,42 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
     append(BronzeTargets.sparkEventLogsTarget) // Not new data only -- date filters handled in function logic
   )
 
+  lazy private[overwatch] val libsSnapshotModule = Module(1007, "Bronze_Libraries_Snapshot", this)
+  lazy private val appendLibsProcess = ETLDefinition(
+    workspace.getClusterLibraries,
+    append(BronzeTargets.libsSnapshotTarget)
+  )
+
+  lazy private[overwatch] val policiesSnapshotModule = Module(1008, "Bronze_Policies_Snapshot", this)
+  lazy private val appendPoliciesProcess = ETLDefinition(
+    workspace.getClusterPolicies,
+    append(BronzeTargets.policiesSnapshotTarget)
+  )
+
+  lazy private[overwatch] val instanceProfileSnapshotModule = Module(1009, "Bronze_Instance_Profile_Snapshot", this)
+  lazy private val appendInstanceProfileProcess = ETLDefinition(
+    workspace.getProfilesDF,
+    append(BronzeTargets.instanceProfileSnapshotTarget)
+  )
+
+  lazy private[overwatch] val tokenSnapshotModule = Module(1010, "Bronze_Token_Snapshot", this)
+  lazy private val appendTokenProcess = ETLDefinition(
+    workspace.getTokens,
+    append(BronzeTargets.tokenSnapshotTarget)
+  )
+
+  lazy private[overwatch] val globalInitScSnapshotModule = Module(1011, "Bronze_Global_Init_Scripts_Snapshot", this)
+  lazy private val appendGlobalInitScProcess = ETLDefinition(
+    workspace.getGlobalInitScripts,
+    append(BronzeTargets.globalInitScSnapshotTarget)
+  )
+
+  lazy private[overwatch] val jobRunsSnapshotModule = Module(1012, "Bronze_Job_Runs_Snapshot", this) // check module number
+  lazy private val appendJobRunsProcess = ETLDefinition(
```

Comment:
I noticed that none of these bronze lands have any transforms which may be fine for most (maybe all of them) but I want to make sure a schema analysis has been completed to ensure there aren't any user defined key/value pairs being landed as structs. If so, we need a bronzeTransform to cleanse. see `cleanseRawClusterSnapDF` func for example on cleansing bronze lands. There also may need to be a schema scrub if there are any spaces or special characters in any of the return schemas.